vhdl_lang/src/syntax/declarative_part.rs

```diff
@@ -332,4 +355,16 @@ var not_a_var: broken;
             )],
         )
     }
+
+    #[test]
+    fn parse_declarative_part_with_disconnection() {
+        let code = Code::new(
+            "\
+constant foo: real := 5.1;
+disconnect my_signal : integer after 42 ms;
+signal bar: std_logic;
+",
+        );
+        code.with_stream_no_diagnostics(parse_declarative_part);
```

Comment:
This is a good test, but it doesn't do a lot yet. It would be better to also check the value of each declaration and see that it matches.

Reply:
I have moved this over into Disconnetion.rs, where I tried to do a lot more in depths tests.